docs/Tutorials/Examples/Private-Network-Example.md

```diff
@@ -4,37 +4,45 @@ description: Quorum dev quickstart tutorial
 
 # Quorum Developer Quickstart tutorial
 
-The Quorum Developer Quickstart uses the Hyperledger Besu Docker image to run a private network of Besu
-nodes managed by Docker Compose.
+The Quorum Developer Quickstart uses the Hyperledger Besu Docker image to run a private
+[IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md) network of Besu nodes managed by Docker Compose.
 
-!!! important
-
-    This tutorial runs a private network suitable for education or demonstration purposes and is
-    not intended for running production networks.
+!!! warning
+    This tutorial runs a private network suitable for education or demonstration purposes
+    and is not intended for running production networks.
 
 ## Prerequisites
 
-* [Docker and Docker-compose](https://docs.docker.com/compose/install/)
-* [Nodejs](https://nodejs.org/en/download/)
-* On Windows:
-    * Windows Subsystem for Linux 2
+* One of the following operating systems is required:
+    * Linux on x86_64 architecture.
```

Comment:
```suggestion
    * Linux on x86_64 architecture
```
This sublist doesn't have periods because the items don't form a complete sentence with the list introduction.

Reply:
Arggg, this is so different than in french...